x-pack/legacy/plugins/maps/common/constants.ts

```diff
@@ -3,6 +3,9 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
+
+// NP Migration
+// Temporarily redundant with x-pack/plugins/maps/common/constants.ts
```

Comment:
I'd re-export here from the np.

something like `export * from foobar/constants`